langchain/package.json

```diff
@@ -58,6 +58,7 @@
     "@typescript-eslint/eslint-plugin": "^5.51.0",
     "@typescript-eslint/parser": "^5.51.0",
     "babel-jest": "^29.4.2",
+    "cheerio": "^1.0.0-rc.12",
```

Comment:
Would be good to put these optional dependencies in `peerDependencies`/`peerDependenciesMeta` so that any consuming packages have ~the same version as we used in dev (and get a warning if not)